src/markdown/tutorial/part-1/01-orientation.md

```diff
@@ -90,7 +90,7 @@ del package.json
 @@ -9,2 +9,3 @@
      EmberENV: {
 +      RAISE_ON_DEPRECATION: true,
-       EXTEND_PROTOTYPES: false,
+       FEATURES: {
```

Comment:
I'm not sure where this change is coming from -- afaict new apps on v5.0.0 still have `EXTEND_PROTOTYPES: false,` as the next line here.

Reply:
strange 🤔 I must have a local dependency issue